### PR TITLE
Add a way to update Markers during the game

### PR DESCRIPTION
**Pull request by AmLibra**
This is going to be used to update the UI on turn switches and to show owned locations and such.

### CODE REVIEW

File: app/src/main/java/com/github/polypoly/app/base/game/location/LocationProperty.kt [hunk 1]
```diff
@@ -17,7 +17,7 @@ data class Zone(
     An enum class representing the different levels of a [LocationProperty].
  */
 enum class PropertyLevel {
-    LEVEL_0, LEVEL_1, LEVEL_2, LEVEL_3, LEVEL_MAX
+    LEVEL_0, LEVEL_1, LEVEL_2, LEVEL_MAX
```
Comment (tamaraaPrs) [inline app/src/main/java/com/github/polypoly/app/base/game/location/LocationProperty.kt:20]: We will have to think about replacing them with our real level names. Why did we delete a level by the way?

File: app/src/main/java/com/github/polypoly/app/ui/map/MapInitializationHelpers.kt [hunk 2]
```diff
@@ -77,33 +81,78 @@ fun initMapView(context: Context): MapView {
  * @param gameViewModel GameViewModel to use for game business logic. Null if no game is going on.
  * @return the marker that was added
  */
-fun addMarkerTo(mapView: MapView, location: LocationProperty, zoneColor: Int,
-                mapViewModel: MapViewModel, gameViewModel: GameViewModel?
+fun addMarkerTo(
```
Comment (tamaraaPrs) [inline app/src/main/java/com/github/polypoly/app/ui/map/MapInitializationHelpers.kt:84]: I find it cleaner to put one parameter per line for readability

File: app/src/main/java/com/github/polypoly/app/ui/map/MapUI.kt [hunk 3]
```diff
@@ -1,40 +1,69 @@
...
+                if (gameViewModel != null)
+                    mapViewModel.updateMapViewState(gameViewModel.getGameData().value!!.inGameLocations)
+                val mapView = createMapView(context, mapViewModel, gameViewModel, mapViewState)
                 mapView
-            }, modifier = Modifier.testTag("map"))
+            }, modifier = Modifier.testTag("map")
+        )
+    }
+
+    private fun createMapView(
```
Comment (tamaraaPrs) [inline app/src/main/java/com/github/polypoly/app/ui/map/MapUI.kt:34]: same, it is more lisible to put one parameter per line

**Review by tamaraaPrs — CHANGES_REQUESTED** [review]
Everything is good except for the very small readability details!

File: app/src/main/java/com/github/polypoly/app/base/game/location/LocationProperty.kt (hunk 1 above)
Comment (AmLibra) [reply app/src/main/java/com/github/polypoly/app/base/game/location/LocationProperty.kt:20]: Reason 1, convenience and making the stars fit on the bitmap, I can make 3 stars fit ish (technically only 2 fit but conveniently 3 shows a full star and 2 star halves on either end)
Reason 2, we will never reach 4 levels anyway.

File: app/src/main/java/com/github/polypoly/app/ui/map/MapInitializationHelpers.kt (hunk 2 above)
Comment (AmLibra) [reply app/src/main/java/com/github/polypoly/app/ui/map/MapInitializationHelpers.kt:84]: done

File: app/src/main/java/com/github/polypoly/app/ui/map/MapUI.kt (hunk 3 above)
Comment (AmLibra) [reply app/src/main/java/com/github/polypoly/app/ui/map/MapUI.kt:34]: done

**Review by tamaraaPrs — APPROVED** [review]

**Pull request merged**